script/tool/test/update_excerpts_command_test.dart

```diff
@@ -109,6 +109,50 @@ void main() {
         ]));
   });
 
+  test('updates example readme when config is present', () async {
+    final RepositoryPackage package = createFakePlugin('a_package', packagesDir,
+        extraFiles: <String>[kReadmeExcerptConfigPath]);
+    final Directory example = getExampleDir(package);
+
+    final List<String> output =
+        await runCapturingPrint(runner, <String>['update-excerpts']);
+
+    expect(
+        processRunner.recordedCalls,
+        containsAll(<ProcessCall>[
+          ProcessCall(
+              'dart',
+              const <String>[
+                'run',
+                'build_runner',
+                'build',
+                '--config',
+                'excerpt',
+                '--output',
+                'excerpts',
+                '--delete-conflicting-outputs',
+              ],
+              example.path),
+          ProcessCall(
+              'dart',
+              const <String>[
+                'run',
+                'code_excerpt_updater',
+                '--write-in-place',
+                '--yaml',
+                '--no-escape-ng-interpolation',
+                'README.md',
+              ],
+              example.path),
+        ]));
+
+    expect(
+        output,
+        containsAllInOrder(<Matcher>[
+          contains('Ran for 1 package(s)'),
+        ]));
+  });
+
```

Comment:
This test looks good, but you're still missing the failure test. It should look like [this one](https://github.com/flutter/packages/blob/56a2f854aa80585e404862e78a1465e949321926/script/tool/test/update_excerpts_command_test.dart#L251-L276), except that there should be [two mock process results for updating instead of one](https://github.com/flutter/packages/blob/56a2f854aa80585e404862e78a1465e949321926/script/tool/test/update_excerpts_command_test.dart#L258-L259), with the first succeeding and the second failing, and then the expectation should be for your new failure case's error message.

Reply:
Forgot to go back and add that one last night, sorry!